hochschule-darmstadt/einander-helfen
# 175/backup enhanced data

closes #175 
To test this you can do the following:
- comment out line 20-31 in main.py
- run main.py once for all required folders to be created
- use the provided testdata.zip
 -- folders inside 'backups' go into 'einander-helfen\etl\data_management\backups', remove previous data
 -- files inside 'enhanced data' go into 'einander-helfen\etl\data_enhancement\data', remove/back up previous data, rename folder '07.01' to current date in the same format if you are testing this on a different day than today

- run main.py again. You'll get an output for the composition of the upload folder. 
- source for dataset1 should be the day when you test it (07.01 for today)
- source for dataset2 should be the day when you test it (07.01 for today)
- source for dataset3 should be the day when you test it (07.01 for today)
- source for dataset4 should be the day when you test it (07.01 for today)
- source for dataset5 should be backup '05.01.2021' as it contains 40 instead of 10 posts
- source for dataset8 should be backup '04.01.2021' as it is unique to that backup
- source for dataset6 should be backup '06.01.2021' as it is unique to that backup
- dataset7 - should be the day when you test it (07.01 for today) as it is unique to the fresh enhanced data

the file names in the output will slightly differ as they have a description of the expected result for ease of use
[testdata.zip](https://github.com/hochschule-darmstadt/einander-helfen/files/5781612/testdata.zip)
